runtime/Go/antlr/utils.go

```diff
@@ -8,7 +8,7 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
-	"math/bits"
+	"sort"
```

Comment:
Unrelated changes. Please remove them from PR (I recommend using a very convenient interactive rebase in IntelliJ for changing local history).

Reply:
that it's just a change when I reverted the Go done by @jcking in this branch when I was playing around.